tests/framing-helper.c

```diff
@@ -1,10 +1,14 @@
-#include <err.h>
 #include <stdio.h>
 #include <stdbool.h>
 #include <stdint.h>
 #include <stdlib.h>
 #include <unistd.h>
+
+#ifdef __MINGW32__
```

Comment:
This is definitely a `_WIN32` issue, but it shouldn't be needed since `htonl`/`ntohl` aren't called on Windows. Normally `#include`ing `winsock2.h` requires you to link the app against winsock2, but I don't think you ran into that since there wasn't actually a call to the winsock2 library. Could you change this by moving the `#include <arpa/inet.h>` inside the `#else` clause below. Also, change `__WIN32__` to `_WIN32` to be consistent (I know, that one was my bad).